src/main/g8/scalafix/input/src/main/scala-3/fix/$repo__Camel$SignificantIndentation.scala

```diff
@@ -0,0 +1,8 @@
+/*
+rule = $repo;format="Camel"$
+*/
+package fix
+
+object $repo;format="Camel"$SignificantIndentation:
```

Comment:
This was initially a way to test that everything works, but since we have now a scripted in `sbt-scalafix`, this is not really required. I do like the fact that it demonstrates the usage of `scala-3` intput/output.